### DISCUSSION TITLE
Wired

### DISCUSSION THREAD

**Discussion by giscus** [aduros/wasm4#431] [Games]
# Wired

Build retro games using WebAssembly for a fantasy console

https://wasm4.org/play/wired

**Comment by ghost** [comment]
Can you make on-screen controls slightly transparent? I can't see the part of the game level :) Thanks!

![](https://user-images.githubusercontent.com/28234322/164204494-6e343fab-2425-4c7c-8afc-9ef78ea266b3.jpeg)

**Comment by ghost** [comment]
cc @aduros :)

**Reply by aduros** [reply]
Hi @procedural, yeah, this is a bug on iPhone, the game should fullscreen when tapped. I made a [commit](https://github.com/aduros/wasm4/commit/a92b8ee4d2c2bb4344619a8360138a3bbc0caf20) that might improve things, could you give it a shot? wasm4.org has been updated.

**Reply by ghost** [reply]
Yep, it fullscreens now, but not entirely, so a small patch of screen is still blocked (iPhone 6s)
![](https://user-images.githubusercontent.com/28234322/165041320-0ff2103a-b8bd-40d6-9ba6-32ac735f08d6.jpeg)

**Reply by aduros** [reply]
Thanks, I don't have an iOS device so I'm shooting in the dark :sweat_smile: If you have time to open a tested PR I'd really appreciate it!

**Reply by ghost** [reply]
If the movement cross was just a tiny bit transparent, just a tiny bit, l'd see what was going on behind it

**Reply by aduros** [reply]
Yeah, but rather than make the gamepad transparent, I'd rather have the game shifted up so it doesn't overlap at all.
